stable/weblate/templates/ingress.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if .Values.ingress.enabled }}
 {{- $fullName := include "weblate.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
Since you support both old and new API versions in the spec below I would also make this line conditional to support both

Reply:
old version support is added